packages/terra-framework-site/tests/jest/TestingComponent.scss

```diff
@@ -0,0 +1,10 @@
+:local {
```

Comment:
Do we need to check in the stats.json file?

Reply:
Ah so that's the giant file it was warning me about. It is checked into core, but I don't think it necessarily should be.

Reply:
We should avoid checking in stats.json